frontend/src/Theme.css

```diff
@@ -29,18 +29,21 @@
 	--action-button-text-colour: var(--main-text-colour);
 	--action-button-disabled-background-colour: #888;
 	--action-button-disabled-text-colour: #111;
+	--action-button-background-colour-hover: #11484b;
 
 	/* level buttons */
 	--level-button-colour: var(--main-text-colour);
 	--level-button-colour-disabled: #777;
 	--level-button-border-colour: var(--accent-border-colour);
 	--level-button-border-colour-disabled: #aaaaaa80;
 	--level-button-background-colour-selected: var(--accent-background-colour);
+	--level-button-background-colour-hover: #11484b;
 
 	/* level banner */
 	--level-banner-border-colour: #bbb;
 	--level-banner-background-colour: #ddd;
 	--level-banner-text-colour: #444;
+	--level-banner-background-colour-hover: #fff;
```

Comment:
small beans, feel free to ignore, but maybe move this underneath the other background colour on line 44